.github/workflows/test-all.yml

```
@@ -206,8 +209,71 @@ jobs:
           PKG=${{ matrix.package }}
           gap -A --quitonbreak -r -c "SetInfoLevel(InfoPackageLoading, PACKAGE_WARNING);res:=TestPackage(\"$PKG\");FORCE_QUIT_GAP(res);"
 
+      - name: "Get job status"
+        id: job-status-default
+        if: ${{ always() }}
+        run: |
+          PKG="${{ matrix.package }}"
+          echo '::set-output name=status::${{ job.status }}'
+
       - name: "Run tests with OnlyNeeded"
+        if: ${{ always() }}
         timeout-minutes: 12
         run: |
           PKG=${{ matrix.package }}
           gap -A --quitonbreak -r -c "SetInfoLevel(InfoPackageLoading, PACKAGE_WARNING);res:=TestPackage(\"$PKG\":OnlyNeeded);FORCE_QUIT_GAP(res);"
+
+      - name: "Get job status with OnlyNeeded"
+        id: job-status-only-needed
+        if: ${{ always() }}
+        run: |
+          PKG="${{ matrix.package }}"
+          echo '::set-output name=status::${{ job.status }}'
+
+      - name: "Create job status json-file"
+        if: ${{ always() }}
+        run: |
+          PKG="${{ matrix.package }}"
+          STATUS_DEFAULT="${{ steps.job-status-default.outputs.status }}"
+          STATUS_ONLY_NEEDED="${{ steps.job-status-only-needed.outputs.status }}"
+          jq --null-input \
+            --arg status_default "${STATUS_DEFAULT}" \
+            --arg status_only_needed "${STATUS_ONLY_NEEDED}" \
+            '{"status_default": $status_default, "status_only_needed": $status_only_needed}' > ${PKG}.json
```

What does using `jq` here gain us over just
```suggestion
          echo "{\"status_default\": \"$STATUS_DEFAULT\", \"status_only_needed\": \"$STATUS_ONLY_NEEDED\"}" > ${PKG}.json
```

---

I initially planned to add more data and thought it was easier to use jq here, so I didn't have to worry about escaping special characters. But now, this is overkill. I commit your suggestion.